Mathlib/Algebra/Group/Invertible/Defs.lean

```diff
@@ -90,51 +90,64 @@ class Invertible [Mul α] [One α] (a : α) : Type u where
   mul_invOf_self : a * invOf = 1
 
 /-- The inverse of an `Invertible` element -/
-prefix:max
-  "⅟" =>-- This notation has the same precedence as `Inv.inv`.
-  Invertible.invOf
+-- This notation has the same precedence as `Inv.inv`.
+prefix:max "⅟" => Invertible.invOf
 
 @[simp]
 theorem invOf_mul_self' [Mul α] [One α] (a : α) {_ : Invertible a} : ⅟ a * a = 1 :=
   Invertible.invOf_mul_self
 
-theorem invOf_mul_self [Mul α] [One α] (a : α) [Invertible a] : ⅟ a * a = 1 :=
-  Invertible.invOf_mul_self
+theorem invOf_mul_self [Mul α] [One α] (a : α) [Invertible a] : ⅟ a * a = 1 := invOf_mul_self' _
 
 @[simp]
 theorem mul_invOf_self' [Mul α] [One α] (a : α) {_ : Invertible a} : a * ⅟ a = 1 :=
   Invertible.mul_invOf_self
 
-theorem mul_invOf_self [Mul α] [One α] (a : α) [Invertible a] : a * ⅟ a = 1 :=
-  Invertible.mul_invOf_self
+theorem mul_invOf_self [Mul α] [One α] (a : α) [Invertible a] : a * ⅟ a = 1 := mul_invOf_self' _
 
 @[simp]
-theorem invOf_mul_self_assoc' [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b := by
+theorem invOf_mul_cancel_left' [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b := by
   rw [← mul_assoc, invOf_mul_self, one_mul]
+example {G} [Group G] (a b : G) : a⁻¹ * (a * b) = b := inv_mul_cancel_left a b
 
-theorem invOf_mul_self_assoc [Monoid α] (a b : α) [Invertible a] : ⅟ a * (a * b) = b := by
-  rw [← mul_assoc, invOf_mul_self, one_mul]
+theorem invOf_mul_cancel_left [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b :=
```

Comment:
If I understand correctly the difference between the primed and non-primed lemmas, is this not supposed to be
```suggestion
theorem invOf_mul_cancel_left [Monoid α] (a b : α) [Invertible a] : ⅟ a * (a * b) = b :=
```
?

Reply:
Good catch